preferences.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Hmmm actually not sure why this is here, but I tink best is not to change the preferences.json file?